applications/nrf_desktop/doc/hid_state.rst

```diff
@@ -3,5 +3,174 @@
 HID state module
 ################
 
-HID state module is responsible for aggregating data from user input sources
-and, depending on the state of the connection, transmitting it to the host.
+The |hid_state| is required for generating reports from input data.
+It is responsible for the following operations:
+
+    * Aggregating data from user input sources.
+    * Tracking state of the HID report subscriptions.
+    * Forming the HID reports.
+    * Transmitting the HID reports to the right subscriber.
+
+Module Events
+*************
+
+<TO DO>
+
+Configuration
+*************
+
+The |hid_state| module is enabled by selecting ``CONFIG_DESKTOP_HID_STATE_ENABLE``.
+This module is optional and turned off by default.
+
+Report expiration
+    With the ``CONFIG_DESKTOP_HID_REPORT_EXPIRATION`` configuration option, you can set the amount of time after which a key will be considered expired.
+    The higher the value, the longer the period after which the nRF Desktop application will recall pressed keys when the connection is established.
```

Comment:
`recall the pressed key` ?
Otherwise, this could suggest recalling the keys after additional delay when the connection occurs.

Reply:
I don't think so. Also these are keys not a one key. After the connection we are sending keys from the queue.

Reply:
Ok, so let's leave it as it is.